tests/testthat/test-stan-rt.R

```diff
@@ -1,61 +1,59 @@
 skip_on_cran()
-suppressMessages(expose_stan_fns("rt.stan", target_dir = system.file("stan/functions", package = "EpiNow2")))
-
 
 # Test update_Rt
 test_that("update_Rt works to produce multiple Rt estimates with a static gaussian process", {
   expect_equal(
-    update_Rt(rep(1, 10), log(1.2), rep(0, 9), rep(10, 0), numeric(0), 0),
```

Comment:
There are no Stan level changes in this PR. I don't know where the change to update_Rt is now so perhaps already on main.

Reply:
Yes, this was in #303 but the tests weren't updated then.